src/dependadotnet/Program.cs

```diff
@@ -1,19 +1,18 @@
-﻿using System;
-using System.IO;
+﻿using System.IO;
 using static System.Console;
 
 namespace dependadot
 {
     class Program
     {
         private static string[] s_patterns = new string[]
-    {".csproj", ".fsproj", ".vbproj"};
+            {".csproj", ".fsproj", ".vbproj"};
 
         static void Main(string[] args)
         {
             var path = string.Empty;
 
-            if (Console.IsInputRedirected)
+            if (IsInputRedirected)

```

Comment:
`Console.` is redundant. The file has using static System.Console at the top. And after removing the `Console.` in this line. `using System;` can be removed, I think.